src/model/product.entity.ts

```diff
@@ -15,4 +15,7 @@ export class Product extends Base {
 
   @Property()
   description: string;
+
+  @Property()
+  views_count: number
```

Comment:
Make sure to follow the naming convention (i.e. camel case)

```suggestion
  viewsCount: number
```